Src/WitsmlExplorer.Api/Workers/AbstractModifyWellWorker.cs

```diff
@@ -0,0 +1,25 @@
+using Witsml.Data;
+using Witsml.Extensions;
+using WitsmlExplorer.Api.Models;
+
+namespace WitsmlExplorer.Api.Workers
+{
+    public abstract class AbstractModifyWellWorker
```

Comment:
As there is nothing actually being inherited I prefer this as a static class how it was before.

Reply:
Agree, I will improve this code in next work item

Reply:
In my opinion this obscures the Query part and deviates further from the established codebase. 
But as commented before, I think it would be a cleaner solution not producing queries from both `Witsml` and `Witsml.Api` . What do you think?

Reply:
Agree to some extent, But for 'Update' queries it has to be constructed in Witsml.Api in order to maintain cleaner code.

I see similar approach in DeleteLogObjectsWorker.CreateRequest

Reply:
So, Now removed the abstract class as @HakonHesselberg commented, and added static method similar to DeleteLogObjectsWorker.

Will think of further optimization/coding practices as we progress